src/main/java/frc/robot/Robot.java

```diff
@@ -103,27 +117,38 @@
       ColorMatch.makeColor(0.361, 0.524, 0.113);
   private final WPI_TalonSRX m_motorControlPanel =
       new WPI_TalonSRX(RoboRIO.kPortMotorControlPanel);
+  private final DoubleSolenoid m_doubleSolenoidControlPanel =
+      new DoubleSolenoid(RoboRIO.kPortDoubleSolenoidForwardControlPanel,
+          RoboRIO.kPortDoubleSolenoidBackwardControlPanel);
   private String m_detectedColorString = "N/A";
   private String m_lastDetectedColorString = "N/A";
   private String m_targetControlPanelColor = "N/A";
   private int m_controlPanelSpinAmount = 0;
 
   // Vision
 
+  /**
+   * Initializes the solenoids.
+   */
+  private void solenoidInit() {
```

Comment:
Okay, this is a little petty, but every time I see the name `solenoidInit()` be called both in `robotInit()` and `disabledInit()`, it confuses me a little. So, if I understand correctly, the purpose of this is to:
- Set all of the solenoids when the robot is powered on.
- Set all of the solenoids when the robot is disabled.

With that in mind, the phrase "Init" feels a little misleading, because for the latter bullet, the solenoids might already have been "initialized" - perhaps they were when being in teleop beforehand. Would it be more accurate to brand this as, say, `solenoidReset()`?

If we were to be really petty, I would make the argument that calling it from `robotInit()` is redundant, because when the robot is powered on, it should already go into `disabledInit()`. That, however, is something that could change the _behavior of the code_, which I am trying to avoid with this review. Perhaps we could file another issue or TODO about that possibility?

Reply:
`solenoidReset()` seems like a more appropriate name than `solenoidInit()` to me as well, and I will make that modification, however this method is called when the robot is both initialized and disabled solely due to the two reasons you listed in bullets.